docs/content/concepts/append-table/append-queue-table.md

```diff
@@ -96,7 +96,7 @@ For streaming reads, records are produced in the following order:
 You can define watermark for reading Paimon tables:
 
 ```sql
-CREATE TABLE T (
+CREATE TABLE t (
```

Comment:
You need to change SELECT query too.

Reply:
@JingsongLi I have completed the modification, can you help me review it?